src/net/http.rs

```diff
@@ -4948,9 +4949,10 @@ mod test {
             let sres = StacksHttpPreamble::consensus_deserialize(&mut data.as_bytes());
             let expected_serrstr = format!("{:?}", &sres);
             test_debug!("Expect '{}'", errstr);
-            assert!(sres.is_err(), expected_serrstr);
+            assert!(sres.is_err(), "{}", expected_serrstr);
```

Comment:
can simplify the assert here as well

Reply:
Addressed in 93a7c393069ca4420433e03338a6534bee19dc8f